README.md

```diff
@@ -16,19 +17,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
 All blocks, block entities and entities are undoable.
 Undo is available only if that change is caused by a player, for example, placing blocks and commands.
 
+
+## Control the game tick (tick)
+Reden provides the `/tick <function>` command to control game ticks conveniently and quickly.
+
+### Function (the `<function>` parameter)
+- `freeze` freeze the game tick
+- `query` display the current TPS information
+- `rate <tps>` adjust TPS (default is `20.0`)
+- `sprint <1d|3d|60s|stop>` speed up the game
+- `step` execute one game tick forward (only works when the game is frozen)
+- `unfreeze` unfreeze the game tick
+- `back` go back to the previous tick  ***Planning***
+
+
 ## RVC
-> WIP
+> Work in progress
 
-RVC, Redstone Version Control, machine history and diff analysis based on git.
+![rvc_list](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/dbab2f3a-0e5e-4103-8003-f283306d62f2)
+
+RVC(Redstone Version Control), provides version history and diff analysis for your machines based on git. Press `R+L` to open RVC screen.
+
+### Update, commit and push
+***Notice: DO NOT use hotkey `R+G` to login to Github [Issue #103](https://github.com/zly2006/reden-is-what-we-made/issues/103)***
+
+If your Reden account has been linked with GitHub(You can link it [here](https://redenmc.com/home/edit)), RVC can upload your machines to GitHub. 
 
-RVC can upload your machines to GitHub if you have bound your GitHub account at https://redenmc.com.
 RVC supports subregions and submodules, so you can use other people's work in your own creations.
 
+
+For the first-time use, you need to log in to your Reden account. After logging in, you will see the following:
```

Comment:
wtf is this, for online Minecraft players they can login automatically and wont see this gui